File: portfolio-site/src/data/Experience.ts
export class Experience {
  role: string;
  company: string;
  date: string;
  description: string;
  skills: string[];

  constructor(
    role: string,
    company: string,
    date: string,
    description: string,
    skills: string[]
  ) {
    this.role = role;
    this.company = company;
    this.date = date;
    this.description = description;
    this.skills = skills;
  }
}

File: portfolio-aspnet/Models/Experience.cs
namespace PortfolioSite.Models;

public class Experience
{
    public string Role { get; set; } = string.Empty;
    public string Company { get; set; } = string.Empty;
    public string Date { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public List<string> Skills { get; set; } = new();
}
